### DISCUSSION TITLE
Join to contributors

### DISCUSSION THREAD

**Discussion by oleksandr-roskovynskyi** [imdhemy/laravel-in-app-purchases#17] [General]
Hello! Will it be possible to join contributors https://packagist.org/packages/imdhemy/laravel-purchases ?

**Comment by imdhemy** [comment]
Contributions are welcome and will be fully credited.

kindly check the [contribution guide](https://github.com/imdhemy/laravel-in-app-purchases/blob/master/CONTRIBUTING.md).
